sphinx/templates/latex/latex.tex_t

```diff
@@ -29,14 +29,14 @@
 <%= fontpkg %>
 <%= fncychap %>
 \usepackage<%= sphinxpkgoptions %>{sphinx}
+\input{sphinxmessage.sty}
 <%= sphinxsetup %>
 <%= fvset %>
 <%= geometry %>
 <%= usepackages %>
 <%= hyperref %>
 <%= contentsname %>
 <%= numfig_format %>
-<%= translatablestrings %>
```

Comment:
perhaps move the `\input{sphinxmessage.sty}` to this location? (no strong optinion)

Reply:
I also don't have strong opinion. So let's move.